app/view/button/StreetViewTool.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * Button to open the Street View tool.
+ *
+ * @class CpsiMapview.view.button.StreetViewTool
+ */
+Ext.define('CpsiMapview.view.button.StreetViewTool', {
+    extend: 'Ext.button.Button',
+    xtype: 'cmv_streetview_tool',
+
+    requires: [
+        'CpsiMapview.controller.button.StreetViewToolController'
```

Comment:
Should this be CpsiMapview.controller.button.StreetViewTool (based on the filename of the controller)?

Reply:
Correct, thanks.